Assets/Animations/Characters/Slime/Slime Hurt Animations/SlimeHurtAnimator.controller

```diff
@@ -0,0 +1,1045 @@
+%YAML 1.1
```

Comment:
Totally fine for this gamejam - instead of multiple animators, using an [AnimatorOverrideController](https://docs.unity3d.com/Manual/AnimatorOverrideController.html) or a [BlendTree](https://docs.unity3d.com/Manual/class-BlendTree.html) would provide a more scalable / modular way of doing this in the future.

Reply:
I was not aware of this feature. I'll make a note of it for the future but probably not gonna update it before the deadline

Reply:
Oh absolutely, don't change anything in the current implementation if it already works.